src/backend/tests/functional_tests/test_crawler_expunger.py

```diff
@@ -29,7 +29,7 @@ def test_expunger_with_open_case(record_with_open_case):
     expunger = Expunger(record_with_open_case)
 
     assert not expunger.run()
-    assert expunger.errors == ['Open cases exist']
+    assert expunger.record.errors == ['Open cases exist']
```

Comment:
Thanks @wittejm I'd suggest in this case the record object should be directly tested.
```suggestion
    assert record_with_open_case.errors == ['Open cases exist']
```